ci-operator/config/openshift/image-based-install-operator/openshift-image-based-install-operator-main.yaml

```diff
@@ -62,6 +76,26 @@ tests:
           cpu: 100m
           memory: 200Mi
     workflow: generic-claim
+- as: e2e-ibio-ztp
```

Comment:
Personally I wouldn't call this "ztp" `e2e-ibio` is fine, otherwise maybe `e2e-ibio-metal` would work to indicate we're using the dev-scripts metal emulation (which I think is when we use "ztp" in assisted)